week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,12 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? choose a random number and times it by 10.
```

Comment:
What is the specific range that number in? Is it between 1 and 1000? -1000 and 1000?